Dixit_Data/CardAccess.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Dixit_Data.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Images;
+
+namespace Dixit_Data
+{
+    /// <summary>
+    /// This class gives the ids of the images 
+    /// and gives an image by its id.
+    /// </summary>
+    class CardAccess : ICardAccess
+    {
+        /// <summary>
+        /// Get the ids of images in a list.
+        /// </summary>
+        /// <returns>ids of images</returns>
+        List<int> ICardAccess.GetIDList()
+        {
+            List<int> ids = new List<int>();
+
+            for(int i = 1; i < 3; ++i)
```

Comment:
Er this is bad. Why hardcode 3 here? Also the id of the card should be simply the name of the file. 

I think you don't need this DLL added here. You could add the card images directly to the project into a folder. Then you could easily read the folder for all the file names and just return them.